shadows/supportv4/src/main/java/org/robolectric/shadows/support/v4/SupportFragmentController.java

```diff
@@ -127,6 +127,16 @@ public void run() {
     return this;
   }
 
+  public SupportFragmentController<F> saveInstanceState(final Bundle outState) {
+    shadowMainLooper.runPaused(new Runnable() {
```

Comment:
Could you add also restoreInstanceState to this PR? I don't see this already and I wonder how would you restore/retrieve data saved in your `saveInstanceState` method.